src/3rdparty/walkontable/test/helpers/common.js

```diff
@@ -1,4 +1,8 @@
 import { normalize, pretty } from './htmlNormalize';
+import svgToAscii from './svgToAscii';
+
+export { default as getSvgPathsRenderer, precalculateStylesAndCommands } from '../../src/svg/svgPathsRenderer';
+export { default as getSvgResizer } from '../../src/svg/svgResizer';
```

Comment:
The helpers should be accessible through `Walkontable.helpers.*` by re-exporting svg helpers or whole module in the `index.js` file. Thanks to that these file path will be able to remove from the `babel.config.js` file which now allows importing these modules separately in E2E test which is forbidden (in E2E we tests only `dist/` files).